src/components/tab-group/ink-bar.ts

```diff
@@ -0,0 +1,23 @@
+import {Directive, Renderer, ElementRef} from '@angular/core';
+
+@Directive({
```

Comment:
You probably also want to add an `@internal` JsDoc; this makes TypeScript ignore it when generating the `.d.ts.`. For example:

```ts
/**
 * This is the description of the ...
 * @internal
 */
```